README.md

```diff
@@ -1,24 +1,30 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Pre-requisites
+Ruby 2.7.x, [yarn](https://yarnpkg.com/)
 
-Things you may want to cover:
 
-* Ruby version
+## Getting started
+```
+$ ruby --version
+# ruby 2.7.x
 
-* System dependencies
+$ git clone https://github.com/ivplus/aggregator.git
+$ cd aggregator
 
-* Configuration
+$ gem install bundler
+$ bundle install
```

Comment:
```suggestion
$ bin/setup
```

I believe this is the recommended way to get the app dependencies (and set up the database).

Reply:
Great, I've made the changes.